pkg/storage/gcs/store.go

```diff
@@ -99,18 +119,23 @@ func (r *gcsReader) ReadAt(p []byte, offset int64) (n int, err error) {
 }
 
 func (g *gcs) Get(ctx context.Context, objectName string) (io.ReadCloser, error) {
+	g.l.Debug("Start Get", zap.String("objectName", objectName))
 	objectReader, err := g.readOnlyClient.Bucket(g.bucket).Object(objectName).NewReader(ctx)
+	g.l.Debug("End Get", zap.String("objectName", objectName))
```

Comment:
Capture the error, same comment for other lines here.

Reply:
ok good catch